_ci/init_tag.sh

```diff
@@ -15,7 +15,7 @@ else
 fi
 
 #in case of release 
-if [ "${TRAVIS_COMMIT_MESSAGE}" = "trigger release" ]; then
+if [ "${TRAVIS_COMMIT_MESSAGE}" = *release* ]; then
```

Comment:
If you want to check if the commit message contains the word "release" I think you need a regex here. Not sure how *release* is going to work.